unfetter-discover/unfetter-ui
# show report, add submitter org when creating reports

adds submitter org to report creation
fixes unfetter-discover/unfetter#848

## To test
* pull this PR
* visit create work product page
* create a new report
* verify submitter org drop down
* select a value
* save a report
* save the work product
* verify in the database 
```
db.getCollection('stix').find({'stix.type': 'report', 'metaProperties.work_products': { $exists: true } })
```
